samples/multiplatform/platform-native/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+import org.jetbrains.kotlin.gradle.plugin.KonanArtifactContainer
+
+apply {
```

Comment:
Better to use one code style. With the `plugins {}` block

Reply:
I have no idea how to move this intro plugins block